src/capi-single-supported/index.js

```diff
@@ -19,7 +19,7 @@ getIframeId()
 .then(() => fetch(`https://api.nextgen.guardianapps.co.uk/commercial/api/capi-single.json?${params}`))
 .then(response => response.json())
 .then(capiData => [addSourceset(capiData.articleImage.sources), populateCard(capiData)])
-.then(([sources, html]) => Promise.all([getWebfonts(['GuardianTextSansWeb', 'GuardianSansWeb']), write(() => container.innerHTML = html)]).then(() => buildImages(sources)))
+.then(([sources, html]) => Promise.all(write(() => container.innerHTML = html)).then(() => buildImages(sources)).then(getWebfonts()))
```

Comment:
Why the change of sequence? Besides, `Promise.all` takes an array of promises as argument.